epochStart/bootstrap/syncValidatorStatus.go

```diff
@@ -111,6 +115,11 @@ func NewSyncValidatorStatus(args ArgsNewSyncValidatorStatus) (*syncValidatorStat
 
 	s.memDB = disabled.CreateMemUnit()
 
+	enableEpochsHandlerNodesCoordinator, err := enablers.NewEnableEpochsHandler(args.EnableEpochsConfig, &timemachine.DisabledEpochNotifier{})
```

Comment:
I think we need to change this pattern as it is very prone to error.
In case someone passes the enableEpochHandler to a subcomponent, it will be difficult to spot in the review that this is a timemachine enableEpochHandler, and it could be changed to view a different epoch than the current one.